grease/README.md

```diff
@@ -0,0 +1,19 @@
+# grease
+
+`grease` is a library for checking properties about binaries using under-constrained symbolic execution.
+
+For more information, see [the blog post introducing GREASE](https://www.galois.com/articles/introducing-grease), or [the documentation](https://galoisinc.github.io/grease).
+
+<img src="doc/logo.png" alt="GREASE logo" width="50%" />
```

Comment:
I think this link is broken

Reply:
Oops, good catch! I hope that GitHub-Flavored Markdown understand what `../doc/logo.png` means...